ibm/service/database/resource_ibm_database.go

```diff
@@ -952,18 +952,18 @@ func ResourceIBMICDValidator() *validate.ResourceValidator {
 }
 
 type Params struct {
-	Version             string `json:"version,omitempty"`
-	KeyProtectKey       string `json:"disk_encryption_key_crn,omitempty"`
-	BackUpEncryptionCRN string `json:"backup_encryption_key_crn,omitempty"`
-	Memory              int    `json:"members_memory_allocation_mb,omitempty"`
-	Disk                int    `json:"members_disk_allocation_mb,omitempty"`
-	CPU                 int    `json:"members_cpu_allocation_count,omitempty"`
-	KeyProtectInstance  string `json:"disk_encryption_instance_crn,omitempty"`
-	ServiceEndpoints    string `json:"service-endpoints,omitempty"`
-	BackupID            string `json:"backup-id,omitempty"`
-	RemoteLeaderID      string `json:"remote_leader_id,omitempty"`
-	PITRDeploymentID    string `json:"point_in_time_recovery_deployment_id,omitempty"`
-	PITRTimeStamp       string `json:"point_in_time_recovery_time,omitempty"`
+	Version             string  `json:"version,omitempty"`
+	KeyProtectKey       string  `json:"disk_encryption_key_crn,omitempty"`
+	BackUpEncryptionCRN string  `json:"backup_encryption_key_crn,omitempty"`
+	Memory              int     `json:"members_memory_allocation_mb,omitempty"`
+	Disk                int     `json:"members_disk_allocation_mb,omitempty"`
+	CPU                 int     `json:"members_cpu_allocation_count,omitempty"`
+	KeyProtectInstance  string  `json:"disk_encryption_instance_crn,omitempty"`
+	ServiceEndpoints    string  `json:"service-endpoints,omitempty"`
+	BackupID            string  `json:"backup-id,omitempty"`
+	RemoteLeaderID      string  `json:"remote_leader_id,omitempty"`
+	PITRDeploymentID    string  `json:"point_in_time_recovery_deployment_id,omitempty"`
+	PITRTimeStamp       *string `json:"point_in_time_recovery_time,omitempty"`
```

Comment:
Would this break backward compatibility?

Reply:
This is the struct that gets marshaled into JSON and is not a change the schema

Reply:
Cool 👍